coursedata/adventures/de.yaml

```
@@ -603,164 +609,195 @@ adventures:
                     ```
 
 
-                start_code: "forward 50\nturn left"
+                start_code: |
+                    forward 50
+                    turn left
             2:
                 story_text: |
-                    In level 1 the turtle could only turn left or right. That is a bit boring!
-                    In level 2 he can point his nose in all directions.
+                    In Level 1 kann sich die Schildkröte nur nach links und rechts drehen.
+                    Das ist etwas langweilig!
+                    In Level 2 kann sie ihre Nase in alle Richtungen strecken.
 
-                    Use 90 to turn a quarter. We call this degrees. A full turn is 360 degrees.
-                    Can you make a figure with this code? Maybe a triangle or a circle?
+                    Nutze 90, um wieder eine viertel Drehung zu machen.
+                    Diese Angabe ist in "Grad (°)".
+                    Eine volle Drehung um sich selbst sind 360° (360 Grad).
+                    Kannst Du mit diesem Code eine bestimmte Form zeichnen?
+                    Ein Dreieck zum Beispiel, oder einen Kreis?
+
+                example_code: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
 
                     ```
-                    print Drawing figures
-                    angle is 90
-                    turn angle
+                    print Zeichne Formen
```

As this is printed out, I think it's more like a title or an announcement / status report to the user, rather than a command for the turtle. Thus maybe:
```suggestion
                    print Formen zeichnen
```
(Of course, "Zeichne formen" could not just be imperative, but also a short form of "Ich (Computer / Programm / Schildkröte) zeichne formen.", so it could be used an announcement / status report, too.)